app/views/application/_navigation.html.slim

```diff
@@ -1,13 +1,10 @@
 - if user_signed_in?
-  ul.nav.navbar-nav
+  ul.navbar-nav.mb-2.mb-lg-0
     = render('navigation_collection_link', model: Task) do
-      i.fa-regular.fa-file-lines style=("color: #006600")
     = render('navigation_collection_link', model: Collection) do
-      i.fa-solid.fa-folder-open style=("color: #000066")
     = render('navigation_collection_link', model: Group) do
-      i.fa-solid.fa-users style=("color: #660000")
     - if current_user.role === 'admin'
       li.nav-item
-        = link_to rails_admin.dashboard_path, 'data-turbolinks': false, class: 'nav-link' do
+        = link_to rails_admin.dashboard_path, 'data-turbolinks': false, class: 'nav-link text-center' do
```

Comment:
Using `text-center` also has an impact on the mobile design:

<img width="579" alt="Bildschirmfoto 2023-12-06 um 23 33 19" src="https://github.com/openHPI/codeharbor/assets/7300329/4f8b8e8f-ca3c-4788-baf7-47baac1022cb">

I am fine with that change, just wanted to point it out.

Reply:
Glad you point that out. You are right! After playing a little bit more with padding, I  think putting it on the left instead of the center will make the menu look much better!